src/common-utils/main.sh

```diff
@@ -385,6 +385,11 @@ else
     user_rc_path="/home/${USERNAME}"
 fi
 
+if [ ! -d "${user_rc_path}" ]; then
```

Comment:
Nit: Can we move this block after L:385? 

https://github.com/devcontainers/features/blob/1a3145d6c52a6b8d319a5639acdcff408f721c67/src/common-utils/main.sh#L385

`/root` will always exists so it's cleaner to add it in there.